src/main/java/Spock.java

```diff
@@ -0,0 +1,62 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class Spock {
+    public static void main(String[] args) {
+        System.out.println("____________________________________________________________");
```

Comment:
Perhaps replace "____________________________________________________________" with a constant to make the code more readable?
```suggestion
        System.out.println(LINE);
```